### PR TITLE
298: Announcements Section

### PR DESCRIPTION
**Pull request by colinnichols**
- added announcements section to home page with Intro Text and Intro Link Text fields
 - added Announcements page with Page Title, Announcement Date and Linked Insight Tag
 - adjusted home page to show Announcements when Intro Text is present which links to the Announcements page
 - added Announcements page with optional Page Title and Announcement Date where the value in Linked Insight Tag will match all Insights posted with that tag

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -5,6 +5,14 @@
   {% include header_home.html %}
 
   <div class="site-content home" id="site_content" aria-label="Content">
+    {% if page['Announcements Section']['Intro Text'].size > 0 %}
```
Comment (freestylebit) [inline _layouts/home.html:8]: I take it we don't have much control over the controller or the model here, right?  Ideally, we want to keep the view simple and just have embedded logic like:

```
{% announcement_section %}
```

or something, so we're not riddling the view layer with a lot of conditional logic.

No need to take action here, but just wanted to put the though out there.

File: _posts/2020-07-06-staying-alive-re-thinking-business-strategy-for-indonesian-msmes.markdown [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
...ting most of the 64.2 million MSMEs in Indonesia. MSMEs play a crucial role in Indonesia's economy and absorb about 97 percent of the total national workforce while account for more than half of the country’s GDP. Therefore, how the MSMEs population can navigate this economic downturn will determine the extent to which jobs and productivity in Indonesia can be preserved.  The Indonesian government has been struggling with how to manage informal MSMEs. According to Statistics Indonesia, about 98 percent of the enterprises operating in Indonesia are micro in size, with small enterprises only representing about 1.2 percent and medium enterprises representing even less around 0.09 percent of total enterprises(1). Despite a decade of business registration and licensing simplification reforms in Indonesia to make it easier and cheaper for enterprises to formalize, Statistics Indonesia still found that 93 percent of the micro and small enterprises operating in the non-agricultural sector in 2018 remained informal. The reforms also did not reduce informal employment, given that in 2018 an estimated 70 percent of the informal workers in Indonesia are employed by micro and small enterprises.
```
Comment (freestylebit) [inline _posts/2020-07-06-staying-alive-re-thinking-business-strategy-for-indonesian-msmes.markdown:13]: Does the template parser translate the single quotes here automatically or should we be using HTML entities?

File: _layouts/home.html (hunk 1 above)
Comment (colinnichols) [reply _layouts/home.html:8]: There is the concept of an `_includes` folder where I can include a section called announcements then put all of that logic in there.  I did that for somethings like `contracts-list` or `project-list`, could do something similar but not sure I care too much.

File: _posts/2020-07-06-staying-alive-re-thinking-business-strategy-for-indonesian-msmes.markdown (hunk 2 above)
Comment (colinnichols) [reply _posts/2020-07-06-staying-alive-re-thinking-business-strategy-for-indonesian-msmes.markdown:13]: I'm pretty sure it translates them.  I'll double check locally.

**Pull request merged**